src/systemd/Makefile.am

```diff
@@ -30,8 +30,8 @@ install-exec-hook::
 	mkdir -p $(DESTDIR)$(sysconfdir)/issue.d
 	ln -sTfr $(DESTDIR)/run/cockpit/motd $(DESTDIR)$(sysconfdir)/issue.d/cockpit.issue
 
-tempconfdir = $(prefix)/lib/tmpfiles.d
-nodist_tempconf_DATA = src/systemd/cockpit-tempfiles.conf
+tmpfilesconfdir = $(prefix)/lib/tmpfiles.d
+nodist_tmpfilesconf_DATA = src/systemd/cockpit-tmpfiles.conf
```

Comment:
In your "Rename cockpit-tempfiles to cockpit-tpmfiles.conf" commit, both the commit message and the actual file rename have "tpmfiles", not "tmpfiles". This also explains the build failure, as there really is no such file.

The commit message also has a "function*n*ality" typo.